libdnf-plugins/python_plugins_loader/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+if(NOT WITH_PYTHON3)
+    return()
+endif()
+
+
+#if(WITH_BINDINGS)
+    if(NOT PYTHON_DESIRED)
+        find_package(PythonInterp REQUIRED)
+    elseif(${PYTHON_DESIRED} STREQUAL "2")
+        find_package(PythonInterp 2 EXACT REQUIRED)
+    elseif(${PYTHON_DESIRED} STREQUAL "3")
+        find_package(PythonInterp 3 EXACT REQUIRED)
+    elseif(EXISTS ${PYTHON_DESIRED})
+        set(PYTHON_EXECUTABLE ${PYTHON_DESIRED})
+        find_package(PythonInterp REQUIRED)
+    else()
+        message(FATAL_ERROR "Invalid PYTHON_DESIRED value: " ${PYTHON_DESIRED})
+    endif()
+    find_package(PythonLibs REQUIRED)
+#endif()
+
+find_package(PythonInstDir)
+include_directories(${PYTHON_INCLUDE_PATH})
```

Comment:
please replace with the following lines:
```
find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
include_directories(${Python3_INCLUDE_DIRS})
```

Reply:
OK. Replaced